python/core/auto_generated/effects/qgsimageoperation.sip.in

```diff
@@ -133,7 +133,7 @@ speed and blur quality.
    alphaOnly is set to ``False``, or ARGB32 if alphaOnly is ``True``
 %End
 
-    static QImage *gaussianBlur( QImage &image, int radius ) /Factory/;
+    static QImage *gaussianBlur( QImage &image, double radius ) /Factory/;
```

Comment:
I don't think there's any need to make this double -- internally all the calculations are integer (and HAVE to be so), so better to leave this as int and instead round off values passed to it.